packages/slice-machine/src/modules/userContext/index.ts

```diff
@@ -33,6 +41,19 @@ export const userHasDoneTheOnboarding = (
   state: SliceMachineStoreType
 ): boolean => state.userContext.isOnboarded;
 
+export const getDismissedUpdateVersion = (
```

Comment:
This selector is only used but the dismissedLatestUpdate, I think you can get ride of it to avoid selector imbrication